nservicebus/upgrades/all-versions.include.md

```diff
@@ -0,0 +1,2095 @@
+### NServiceBus
+
+<table>
+  <tr>
+    <th>Version</th>
+    <th>Released</th>
+    <th>Supported until</th>
```

Comment:
@adamralph I agreed with that when we discussed it during development, but as I was transitioning the change to the Docs repo, I reran the script. Because NSB.RavenDB 3.2 was released, there was a change, and I noticed that because the table layout is so spread out vertically, I couldn't even tell from the diff hunk which package the change was for without going into a full diff tool.

Also, using MD tables would make it trivially easy to center a column, like "Supported until" which I think needs centering.

So I'm flipping to supporting markdown tables. Padded is nice, but I don't think we need to do anything fancy like figure out what is longest and then dynamically pad. I think it can just be hardcoded widths.